ape_infura/providers.py

```diff
@@ -24,6 +24,9 @@ def disconnect(self):
     def update_settings(self, new_settings: dict):
         pass
 
+    def chain_id(self) -> int:
+        return self, self._web3.eth.chain_id  # type: ignore
```

Comment:
Sorry, I made a mistake before
```suggestion
        return self._web3.eth.chain_id
```